src/components/ConfirmModal.js

```diff
@@ -0,0 +1,40 @@
+import Modal from "./Modal.js";
+
+export default class ConfirmModal extends Modal {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.